lib/much-rails/action/unprocessable_entity_result.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "much-rails/action/base_result"
+
+# MuchRails::Action::UnprocessableEntityResult is a result returned by
+# calling a view action that is not valid. It returns JSON with the validation
+# details.
+#
+# This result is only returned if, after validating the action, there are
+# errors. Most commonly this occurs when validating form submission params.
+module MuchRails; end
+module MuchRails::Action; end
+class MuchRails::Action::UnprocessableEntityResult <
+        MuchRails::Action::BaseResult
+
```

Comment:
why do you keep adding a blank line between class opening and `attr_reader`?

Reply:
Must have done it on the 1st one and followed the bad pattern, no real reason.